cfme/fixtures/pytest_selenium.py

```diff
@@ -594,8 +594,8 @@ def click(loc, wait_ajax=True, no_custom_handler=False):
     # and then click on current mouse position
     ActionChains(browser()).click().perform()
     # -> using this approach, we don't check if we clicked a specific element
-    if wait_ajax:
-        wait_for_ajax()
+    # if wait_ajax:
```

Comment:
Basically, we now probably need @mfalesni to step in and help us create a new browser model for SSUI. Another WT layer perhaps too.